src/remote.js

```diff
@@ -259,20 +278,20 @@
                       util.err(error);
                     }
                   )
-                },
-                function(error) {
-                  util.err(error);
                 }
-              )
+              },
+              function(error) {
+                util.err("Unable to build this project. " + error);
```

Comment:
No, my point is that the promise have different types (Error and stringified JSON) when rejecting the flow. And, the raw stringified JSON is output to the user. I don't think it is a good idea at all.